### PR TITLE
Architecture update

### PR DESCRIPTION
**Pull request by Tabrarian**
Adding the Forseti architecture framework as discussed. Apologies for the delay, I didn't have the image available while working remotely the last couple days!

### CODE REVIEW

File: _docs/_latest/concepts/architecture.md [hunk 1]
```diff
@@ -1,8 +1,43 @@
...
+    Forseti modules.
+ 3. Scanner compares the data collected by Inventory to the policy rules you
+    set.
+ 4. **what's happening with the Cloud SQL to GCP connection here?**
+ 5. Notifier sends Scanner findings to one or more of the following channels you
+    configure: Cloud Storage, SendGrid, and Slack.
+ 6. You use Explain to check your Cloud IAM policies.
+ 7. Enforcer uses Google Cloud APIs to make sure policies match your desired
+    state.
+ 8. **what's happening with the CLI to GRPC Server connection here?**
```
Comment (mirons-google) [inline _docs/_latest/concepts/architecture.md]: This intended to state that the CLI is using GRPC to talk to a server on a different host to exchange the information necessary to answer the query the user issued on the CLI.

File: _docs/_latest/concepts/architecture.md [hunk 2]
```diff
@@ -1,8 +1,43 @@
...
+The image below shows how data flows through Forseti:
+
+![Forseti module architecture diagram](/images/docs/concepts/forseti-architecture.png)
+
+ 1. Inventory collects information about your GCP resources and G Suite Groups.
+ 2. Inventory stores information in Cloud SQL for your review and use by other
+    Forseti modules.
+ 3. Scanner compares the data collected by Inventory to the policy rules you
+    set.
+ 4. **what's happening with the Cloud SQL to GCP connection here?**
```
Comment (mirons-google) [inline _docs/_latest/concepts/architecture.md]: This line should not be here.

**Comment by Tabrarian** [comment]
Updated with the new image and addressed comments. The image shows as under images/docs/concepts/ for me, can you take another look? Thanks!

File: _docs/_latest/concepts/architecture.md [hunk 3]
```diff
@@ -1,8 +1,42 @@
...
+![Forseti module architecture diagram](/images/docs/concepts/forseti-architecture.png)
+
+ 1. Inventory collects information about your GCP resources and G Suite Groups.
+ 2. Inventory stores information in Cloud SQL for your review and use by other
+    Forseti modules.
+ 3. Scanner compares the data collected by Inventory to the policy rules you
+    set.
+ 4. Notifier sends Scanner findings to one or more of the following channels you
+    configure: Cloud Storage, SendGrid, and Slack.
+ 5. You use Explain to check your Cloud IAM policies.
```
Comment (mirons-google) [inline _docs/_latest/concepts/architecture.md]: You use Explain to query and understand your Cloud IAM policies.

File: _docs/_latest/concepts/architecture.md [hunk 4]
```diff
@@ -1,8 +1,42 @@
...
 
 # {{ page.title }}
 
-TODO
+This page describes how Forseti is built, and how the components of Forseti fit
+together.
+
+Forseti gives you tools to understand all the resources you have in Google Cloud
+Platform (GCP). The core Forseti modules work together to provide complete
+information so you can take action to conserve resources, reduce cost, and
```
Comment (mirons-google) [inline _docs/_latest/concepts/architecture.md]: ...action to secure and minimize security risks.

File: _docs/_latest/concepts/architecture.md [hunk 5]
```diff
@@ -1,8 +1,42 @@
 ---
-title: Architecture
-order: 100
+title: Forseti Architecture
```
Comment (mirons-google) [inline _docs/_latest/concepts/architecture.md]: I think this is Okay being just Architecture.

Comment (Tabrarian) [reply _docs/_latest/concepts/architecture.md]: Done.

File: _docs/_latest/concepts/architecture.md (hunk 4 above)
Comment (mirons-google) [reply _docs/_latest/concepts/architecture.md]: SGTM

**Pull request merged**